data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -22,12 +22,12 @@
 
         <createTable tableName="utxo_transaction_status">
             <column name="transaction_id" type="VARCHAR(160)">
-                <constraints nullable="false" foreignKeyName="fk_utxo_transaction_status" references="utxo_transaction(id)"/>
+                <constraints nullable="false" primaryKey="true" foreignKeyName="fk_utxo_transaction_status" references="utxo_transaction(id)"/>
             </column>
             <column name="status" type="VARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="created" type="TIMESTAMP">
+            <column name="updated" type="TIMESTAMP">
```

Comment:
This is convenient for me since I had an `UPDATE` statement in my own PR so this means I don't need to change it too much. That being said, I need to make it update the `updated` column. I'll raise a ticket for that.